docs/governance/index.md

```diff
@@ -36,3 +36,110 @@ All proposals can be found on the Hummingbot Foundation Snapshot: https://snapsh
 
 Voting is expected to commence in mid-to-late January 2022 with the Hummingbot 1.0 release. 
 
+## Pull Request Proposal (PRP)
```

Comment:
Remove one # to differentiate the section

Reply:
# is the page-level heading. I suggest using ## for this one and ### for the sections under this.